lib/Resource.php

```diff
@@ -199,10 +199,10 @@ public function htmlLink($text = null, $options = array())
                 );
             }
 
-            $html .= " ".htmlspecialchars($key)."=\"".
-                         htmlspecialchars($value)."\"";
+            $html .= " ".htmlspecialchars($key, ENT_COMPAT)."=\"".
```

Comment:
See https://php.watch/versions/8.1/html-entity-default-value-changes - adding ENT_COMPAT preserves the current behaviour.